src/vma/sock/sock-redirect.cpp

```diff
@@ -595,8 +597,10 @@ int connect(int __fd, const struct sockaddr *__to, socklen_t __tolen)
 	if (!orig_os_api.connect) get_orig_funcs();
 	BULLSEYE_EXCLUDE_BLOCK_END
 
+#if !defined(VMA_OPTIMIZE_LOG)
 	char buf[256];
 	srdr_logdbg_entry("fd=%d, %s", __fd, sprintf_sockaddr(buf, 256, __to, __tolen));
+#endif /* VMA_OPTIMIZE_LOG */
```

Comment:
Suggesting:
```code
  	char buf[256];
  	NOT_IN_USE(buf); /* to suppress warning in case VMA_OPTIMIZE_LOG */
  	srdr_logdbg_entry("fd=%d, %s", __fd, sprintf_sockaddr(buf, 256, __to, __tolen));
```